Assets/UnityShell/Editor/Scripts/UnityShellEditorWindow.cs

```diff
@@ -12,15 +12,24 @@ static class Styles
 		{
 			public static readonly GUIStyle textAreaStyle;
 
+			// Default background Color(0.76f, 0.76f, 0.76f)
+			private static readonly Color BgColorLightSkin = new Color(0.87f, 0.87f, 0.87f);
+			// Default background Color(0.22f, 0.22f, 0.22f)
+			private static readonly Color BgColorDarkSkin = new Color(0.11f, 0.11f, 0.11f);

```

Comment:
So I tried this out and it's quite a bit darker for the dark skin (with Texture2D in linear space), was that your intent?

Reply:
Ah right, I remembered it to be quite dark, but actually didn't check. My thought was to get it the same amount darker as the light skin bg got lighter. The light skin bg has the same color as a script preview in the inspector btw.

I changed it back to your original color. The number is different due to linear color space, but should be the same again :)